tools/depcheck/pkg/cmd/trace/pkg.go

```diff
@@ -91,7 +91,10 @@ func BuildGraph(packages *PackageList, excludes []string) (*depgraph.MutableDire
 
 			to, exists := g.NodeByName(dependency)
 			if !exists {
-				return nil, fmt.Errorf("expected child node for dependency %q was not found in graph", dependency)
+				// if a package imports a dependency that we did not visit
```

Comment:
this was supposed to end up in a separate pull. did I miss it?

Reply:
Opened a pull here to begin addressing this https://github.com/openshift/origin/pull/18606

Reply:
> Opened a pull here to begin addressing this #18606

whoa, that pull is big.  I was expecting it to be 5 lines long.